src/main/java/com/ioet/bpm/peopletime/boundaries/TimeEventController.java

```diff
@@ -0,0 +1,108 @@
+package com.ioet.bpm.peopletime.boundaries;
+
+import com.ioet.bpm.peopletime.domain.TimeEvent;
+import com.ioet.bpm.peopletime.domain.TimeTemplate;
+import com.ioet.bpm.peopletime.repositories.TimeEventRepository;
+import com.ioet.bpm.peopletime.repositories.TimeTemplateRepository;
+import com.ioet.bpm.peopletime.services.TimeEventService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Date;
+import java.util.Optional;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/time-events")
+@Api(value = "/time-events", description = "Manage Time Events", produces = "application/json")
+public class TimeEventController {
```

Comment:
You want to have two separate packages for the controllers you have created:
For example

com.ioet.bpm.peopletime.timeevents.boundaries

and for your other controller
com.ioet.bpm.peopletime.timetemplates.boundaries

And then separate the logic in each package.

Reply:
In that way, your services, domains and repositories have to be moved to those other packages as well.